libraries/SITL/SIM_JSON.cpp

```diff
@@ -103,34 +102,20 @@ void JSON::set_interface_ports(const char* address, const int port_in, const int
 */
 void JSON::output_servos(const struct sitl_input &input)
 {
-    size_t pkt_size = 0;
-    ssize_t send_ret = -1;
-    if (SRV_Channels::have_32_channels()) {
-      servo_packet_32 pkt;
```

Comment:
Doesn't this break our existing support for 32 servos?

Reply:
dammit, sorry about that, fixed